### PR TITLE
Auth components in BluePrint.

### PR DESCRIPTION
**Pull request by sealionkat**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -15,6 +15,8 @@ import {updateLocation} from './location/LocationActions';
 
 import '../css/sass/main.scss';
 
+import '@blueprintjs/core/dist/blueprint.css';
```
Comment (thealmightyone) [inline src/index.js]: this was removed by @stepek in 65a1199f6cdbdd723a22d19cfb2738b6f377f08d

Comment (sealionkat) [inline src/index.js]: Now should be OK.

File: src/auth/components/LoginBp.jsx [hunk 2]
```diff
@@ -1,6 +1,15 @@
 import React, {Component, PropTypes} from 'react';
 import {Button} from '@blueprintjs/core';
 
+const inputStyle = {
+  width: '100%'
```
Comment (stepek) [inline src/auth/components/LoginBp.jsx]: Could you move css to sccs file?

File: src/auth/Auth.jsx [hunk 3]
```diff
@@ -12,7 +12,7 @@ const style = {
   maxWidth: 450,
```
Comment (stepek) [inline src/auth/Auth.jsx]: Could you move css to sccs file?

File: src/auth/components/LoginBp.jsx [hunk 4]
```diff
@@ -1,6 +1,15 @@
...
+};
+
+const submitStyle = {
```
Comment (stepek) [inline src/auth/components/LoginBp.jsx]: Could you move css to sccs file?

File: src/auth/Auth.jsx [hunk 5]
```diff
@@ -3,9 +3,9 @@ import {connect} from 'react-redux';
 
 import {login, selectTenant, fetchTokenData} from './AuthActions';
 import {resetErrorMessage} from './../error/ErrorActions';
-import SelectTenant from './components/SelectTenant';
 import Alert from '../components/Alert';
 import Login from './components/Login';
+import SelectTenantBp from './components/SelectTenantBp';
 
 const style = {
```
Comment (stepek) [inline src/auth/Auth.jsx]: Could you move css to sccs file?

File: src/auth/components/SelectTenantBp.jsx [hunk 6]
```diff
@@ -0,0 +1,72 @@
+import React, {Component, PropTypes} from 'react';
+import {Button} from '@blueprintjs/core';
+
+const submitStyle = {
```
Comment (stepek) [inline src/auth/components/SelectTenantBp.jsx]: Could you move css to sccs file?

**Pull request merged**